.github/workflows/rust.yaml

```
@@ -0,0 +1,111 @@
+      - uses: actions/cache/save@v4
+        id: cache-test-save
+        if: (success() || failure()) && steps.cache-coverage-restore.outputs.cache-hit != 'true'
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
```

actionlint (1.7.4)

input "restore-keys" is not defined in action "actions/cache/save@v4". available inputs are "enableCrossOsArchive", "key", "path", "upload-chunk-size"

(action)